content/api/introduction/index.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Horizon API Reference
 order: 0
+link: https://github.com/stellar/new-docs/blob/master/content/api/introduction/index.mdx
```

Comment:
Please add "link" in frontmatter if we want to link it to a relevant github link @kylemccollom

Reply:
can we set up this edit button to link to the relevant `.mdx` file? for example, the edit button on https://stellar-documentation.netlify.com/api/resources/ledgers/object/ would link to https://github.com/stellar/new-docs/blob/master/content/api/resources/ledgers/object.mdx. so each page's edit button would auto link to the file that contains the contents from which the page was built.

Reply:
smart

Reply:
Updated